README.md

```diff
@@ -89,15 +89,14 @@ import { createStore } from 'redux'
  * follows a different convention (such as function maps) if it makes sense for your
  * project.
  */
+ 
+actionTypes = {
+  "INCREMENT": function(state){ return state + 1 },
+  "DECREMENT": function(state){ return state - 1}
+}
+ 
 function counter(state = 0, action) {
-  switch (action.type) {
-  case 'INCREMENT':
-    return state + 1
-  case 'DECREMENT':
-    return state - 1
-  default:
-    return state
-  }
+  return actionTypes(action.type)(state);
```

Comment:
You are missing the default branch where the reducer doesn't match any action type and returns the original state.

There's also a typo here. You want square brackets: `actionTypes[action.type]`.

But ultimately this makes the code harder to understand, and since this is an example that should be the first priority.